packages/components/src/components/loaders/Progress.tsx

```diff
@@ -6,8 +6,8 @@ const Wrapper = styled.div`
     width: 100%;
 `;
 
-const Value = styled.div<{ $max: number; $value: number }>`
-    background: ${({ theme }) => theme.BG_GREEN};
+const Value = styled.div<{ $max: number; $value: number; isRed: boolean }>`
+    background: ${({ theme, isRed }) => (isRed ? theme.BG_RED : theme.BG_GREEN)};
```

Comment:
nitpick: I'd rather pass "background" property which could be any color. 
maybe cc @tomasklim or @dahaca in case they want to add something

Reply:
I got inspiration from `packages/components/src/components/buttons/Button/index.tsx` where it is using a property called `isWhite` but I do not have hard opinions on that, so happy to listen other opinions.

Reply:
me neither, unless somebody from @matejkriz insists on changing this I am OK with your current solution, especially, as you are saying, it is consistent with other components

Reply:
as long as there's only one switch property on the component, I'm ok with that